src/enum/tabNameState.ts

```diff
@@ -0,0 +1,16 @@
+export enum TabName {
+  NewsFeed = 'NewsFeed',
+  Explorer = 'Explorer',
+  Timeline = 'Timeline',
+  Gallery = 'Gallery',
+  Communities = 'Communities',
+  Accounts = 'Accounts',
+}
+
+export type TabNameSubset =
```

Comment:
imo, we can use enum as a type so we might be need this

Reply:
Hi @bonanaaaaaa  Can we fix this later? I cannot move forward by updating old code xD

Reply:
Hi @bonanaaaaaa Just updated with type. Thanks for helping me.